pages/common/swagger-codegen.md

```diff
@@ -0,0 +1,19 @@
+# swagger-codegen
+
+> Generate code and documentation for your REST api from a OpenAPI/swagger definition
+
+- Generate from a file or an url
```

Comment:
Does it generate both code and documentation ? Or only one of those ? We should specify that to be clear.

Reply:
both